lib/jnpr/junos/device.py

```diff
@@ -940,6 +976,22 @@ def __init__(self, *vargs, **kvargs):
     # -----------------------------------------------------------------------
     # Basic device methods
     # -----------------------------------------------------------------------
+    @property
+    def connected(self):
+        return self._connected
+
+    @connected.setter
+    def connected(self, value):
+        if value in [True, False]:
+            self._connected = value
```

Comment:
1.  I have the property setter there specifically to do type enforcement.  Otherwise, this would have been a member.  
2.  Yes, I suppose you're right.  Let me make some changes for htat.